### ISSUE TITLE
`plotIndividualTimeProfile` - wrong legend symbols

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/OSPSuite-R#1206]
```
manyObsDC <- readRDS(getTestDataFilePath("manyObsDC"))
manyObsDC$removeGroupAssignment(names = "Vergin 1995.Iv")

dataSet3 <- DataSet$new(name = "Data set 3")
dataSet3$setValues(xValues = 1, yValues = 1, yErrorValues = 1)
dataSet4 <- DataSet$new(name = "Data set 4")
dataSet4$setValues(xValues = 1, yValues = 1, yErrorValues = 1)

manyObsDC$addDataSets(c(dataSet3, dataSet4))

plotIndividualTimeProfile(manyObsDC)
```

![image](https://user-images.githubusercontent.com/25903040/217775889-b9e163c6-d065-4ba5-99d1-84d942834f78.png)

**Comment by Felixmil** [comment]
It is actually linked to the default dataMapping created by `plotIndividualTimeProfile()`.
shapes and color depict two different variable as we can see here:

https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/b438bc7dda2da00503c38e368faf9cd9f49b63cd/R/plot-individual-time-profile.R#L118-L119

Indeed, with the sample code you provided we get these combination for `name` and `group`:

|name                |group              |
|:-------------------|:------------------|
|Data set 3          |Data set 3         |
|Data set 4          |Data set 4         |
|Laskin 1982.Group C |Aciclovir observed |
|Vergin 1995.Iv      |Vergin 1995.Iv     |

So the question is: **should shape always describe the same information than color (i.e `group`) ?**

If yes, the "fix" is simple:

```r
  color <- fill <- shape <- "group"
  linetype <-  "name"
# ...
profilePlot <- profilePlot + ggplot2::guides(linetype = "none")
```

![image](https://user-images.githubusercontent.com/34234913/228180733-098a2a65-4787-4b8f-813a-e0fb78609c8d.png)

**Comment by PavelBal** [comment]
>  should shape always describe the same information than color (i.e group) ?

No, not necessarily.

If multiple data sets are assigned to the same group, they should all have the same color but different symbols in the plot. In the legend, they get only one entry with the name of the group, the color of the group, and the symbol of the first data set within this group.

```
ds1 <- DataSet$new("ds1")
ds1$setValues(c(1,2), c(1,2))

ds2 <- DataSet$new("ds2")
ds2$setValues(c(1,2), c(3,4))

dc <- DataCombined$new()

dc$addDataSets(c(ds1, ds2), groups = c("Group1", "Group1"))

plotIndividualTimeProfile(dc)
```

![image](https://user-images.githubusercontent.com/25903040/229092320-439ead32-de1b-4218-9810-240943a2f5a9.png)

**Comment by Felixmil** [comment]
Then we would have something like this with two separate legends: one for color and one for shape.
![image](https://user-images.githubusercontent.com/34234913/229096893-e721a75d-0a24-459b-8ce2-a23156ad3632.png)

**Comment by PavelBal** [comment]
Can't we override legend entries manually? So the `plot...()` functions in `ospsuite` would calculate legend entries based on the `DataMapping` and pass it to TLF directly, without letting `ggplot2` setting legend automatically?

The proposed solution with two separate legends is not optimal. Imagine we have multiple individual data sets mapped to one group, then the legend becomes extremely full.

**Comment by Felixmil** [comment]
Would this be the desired output from your last exemple ?
![image](https://user-images.githubusercontent.com/34234913/229119571-58f9fa9c-a416-4f88-a0c7-4ad80b495853.png)

**Comment by PavelBal** [comment]
No, the last example is rendered correctly, only one legend entry.

**Comment by PavelBal** [comment]
Here another example that is rendered wrong:

library("ospsuite")
ds1 <- DataSet$new("ds1")
ds1$setValues(c(1,2), c(1,2))

ds2 <- DataSet$new("ds2")
ds2$setValues(c(1,2), c(3,4))

dc <- DataCombined$new()

dc$addDataSets(c(ds1, ds2))

```
plotIndividualTimeProfile(dc)library("ospsuite")
ds1 <- DataSet$new("ds1")
ds1$setValues(c(1,2), c(1,2))

ds2 <- DataSet$new("ds2")
ds2$setValues(c(1,2), c(3,4))

dc <- DataCombined$new()

dc$addDataSets(c(ds1, ds2))

plotIndividualTimeProfile(dc)
```

![image](https://user-images.githubusercontent.com/25903040/229499338-013bf4e8-c0eb-4fad-8f83-402a3a791a54.png)

The two data sets are not grouped, and we get two different legend entries, which is correct. However, both legend entries use the same shape, while the data sets are plotted using different shapes (expected behavior).
